src/app/query/bitcoin/address/utxos-by-address.hooks.ts

```diff
@@ -22,12 +22,25 @@ export function filterUtxosWithInscriptions(
 }
 
 /**
- * Warning: ⚠️ These are **all** UTXOs, including Stamped and Inscribed UTXOs.
- * You should probably use `useCurrentNativeSegwitAccountSpendableUtxos` instead.
+ * Warning: ⚠️ To avoid spending inscriptions, when using UTXOs
+ * we set `shouldFilterInscriptionUtxos` to true
  */
-export function useCurrentNativeSegwitUtxos() {
+export function useCurrentNativeSegwitUtxos(shouldFilterInscriptionUtxos = true) {
```

Comment:
I like flag idea, removes scope for using wrong hook, while still allowing an escape when needed

But, doing as an object is requirement IMO for readability:

```ts
useCurrentNativeSegwitUtxos(false) // no idea what it does

useCurrentNativeSegwitUtxos({ filterUtxos: false }) // better
```

Reply:
yeah, this is better, refactored